src/main/java/frc/robot/subsystems/conveyor/commands/CargoIn.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class CargoIn extends CommandBase {
```

Comment:
Maybe rename to `conveyCargoIn`, or better, `ConveyCargoUp`, because commands' names should start with a verb.

Reply:
Actually I'm not sure what the difference is between `CargoIn` And `Convey`